CMakeLists.txt

```diff
@@ -0,0 +1,120 @@
+cmake_minimum_required(VERSION 3.0.0)
+
+set(ProjectName "Lamina")
+
+project("${ProjectName}" VERSION 1.0.0)
+set(CMAKE_CXX_STANDARD 20)
+
+set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
```

Comment:
Committing commented code is seen as sloppy and should be avoided as much as possible. If you're using the comment as a switch, consider just using a normal if-statement in cmake with a flag near the top of the file. That way, it looks intentional and clear to the reader.